examples/llsec/ipv6-demos/simple-udp-rpl/project-conf.h

```diff
@@ -0,0 +1,78 @@
+#ifndef PROJECT_SIMPLE_UDP_CONF_H_
+#define PROJECT_SIMPLE_UDP_CONF_H_
+
+/* set CCA thresholds */
+#undef CC2420_CONF_CCA_THRESH
+#define CC2420_CONF_CCA_THRESH -25
+#undef CC2538_RF_CONF_CCA_THRES
```

Comment:
Does this changes anything? checking the `cc2538-rf` there is:

````
#ifdef CC2538_RF_CONF_CCA_THRES
#define CC2538_RF_CCA_THRES CC2538_RF_CONF_CCA_THRES
#else
#define CC2538_RF_CCA_THRES CCA_THRES_USER_GUIDE /** User guide recommendation */
#endif /* CC2538_RF_CONF_CCA_THRES */
````

But in the actual code is pretty much hard-coded

````
REG(RFCORE_XREG_CCACTRL0) = CC2538_RF_CCA_THRES_USER_GUIDE;
````